scripts/base/files/pe/main.zeek

```diff
@@ -60,13 +60,13 @@ const pe_mime_types = { "application/x-dosexec" };
 event zeek_init() &priority=5
 	{
 	Files::register_for_mime_types(Files::ANALYZER_PE, pe_mime_types);
-	Log::create_stream(LOG, [$columns=Info, $ev=log_pe, $path="pe", $policy=log_policy]);
+	Log::create_stream(LOG, Log::Stream($columns=Info, $ev=log_pe, $path="pe", $policy=log_policy));
```

Comment:
I understand the motivation, but IMO there's also a style/history and maybe eco-system consistency aspect to this.

I cannot picture the performance characteristics of the record construction making a difference for `Log::create_stream()` and instead it looks a bit alien at first and this is changing a long established pattern. The `[...]` looked a bit like key-word args, which isn't exactly true, but looked nice to my eyes.

I can see arguments for always using the explicit version looking more consistent, but that doesn't seem worth the churn (totally IMO and hopefully someone else chimes in). The performance argument definitely doesn't work for `Log::create_stream()`.